experimental/LieAlgebras/src/AbstractLieAlgebra.jl

```diff
@@ -162,10 +128,37 @@ end
 #
 ###############################################################################
 
-function is_abelian(L::AbstractLieAlgebra)
+@attr function is_abelian(L::AbstractLieAlgebra)
   return all(e -> iszero(length(e)), _struct_consts(L))
```

Comment:
Unrelated, just wondering: would this also work?
```suggestion
  return all(is_empty, _struct_consts(L))
```

Reply:
Yes, this should work, although I don't like that `is_empty` for `SRow` is using the abstract iterator fallback from julia base, and this name doesn't survey what I wanna test here. (`SRow`s iterate only their non-zero entries).

But the following should work and is indeed nicer than the current version.
```suggestion
  return all(iszero, _struct_consts(L))
```